elements/bulk/nuxeo-edit-documents-button.js

```diff
@@ -226,9 +240,15 @@ class NuxeoEditDocumentsButton extends mixinBehaviors([I18nBehavior, FiltersBeha
           // if the bulk widget is not tagged with a `boundPath`, it's safe to assume it has no value
           const value = el.boundPath ? bulkLayout.get(el.boundPath) : null;
           // check if trying to replace with empty value
-          const replacingWithEmptyValue = el.updateMode === 'replace' && this._isValueEmpty(value);
+          const replacingWithEmptyValue =
+            (el.updateMode === 'replace' || el.updateMode === 'addValues') && this._isValueEmpty(value);
           if (replacingWithEmptyValue) {
-            el._setError(this.i18n('bulkWidget.error.replaceWithEmpty'));
+            if (el.updateMode === 'replace') {
+              el._setError(this.i18n('bulkWidget.error.replaceWithEmpty'));
+            }
+            if (el.updateMode === 'addValues') {
+              el._setError(this.i18n('bulkWidget.error.addValuesWithEmpty'));
+            }
```

Comment:
nit: could be made smaller with something along these lines
```suggestion
            el._setError(this.i18n(`bulkWidget.error.${
              el.updateMode === 'replace' ? 'replaceWithEmpty' : 'addValuesWithEmpty'
            }`));
```
or even
```suggestion
            el._setError(this.i18n(`bulkWidget.error.${el.updateMode}WithEmpty`));
```

Reply:
I will push changes with your first suggestion because I think it improves readability. Please let me know if you guys agree.